source/iterator/iterator.go

```diff
@@ -199,26 +203,26 @@ func (iter *Iterator) switchToCDCIterator(ctx context.Context) error {
 	return nil
 }
 
-// checkIfTableExists checks if table exist.
-func checkIfTableExists(ctx context.Context, tx *sql.Tx, table string) (bool, error) {
-	rows, err := tx.QueryContext(ctx, fmt.Sprintf(queryIfTableExists, table))
+func (iter *Iterator) populateKeyColumns(ctx context.Context) error {
+	if len(iter.keyColumns) != 0 {
+		return nil
+	}
+
+	iter.keyColumns = []string{iter.orderingColumn}
```

Comment:
ordering column must be in keys if don't have key from config and keys from table. please change it

Reply:
but where do you reassign  it ?

Reply:
Here:
https://github.com/conduitio-labs/conduit-connector-oracle/blob/feature/source/key-column/source/iterator/iterator.go#L225

Reply:
we created slice with ordering column value on line 211
then if we have primary keys we add primary key  to this slice using append but ordering column  was already on this slice and we will have slice with ordering column

Reply:
Updated 👍